chain/consensus/mir/manager.go

```diff
@@ -135,30 +137,27 @@ func NewManager(ctx context.Context, addr address.Address, api v1api.FullNode) (
 	net.Connect(ctx)
 	log.Debug("Mir network transport connected")
 
-	reqStore := reqstore.NewVolatileRequestStore()
-
 	// Instantiate the ISS protocol module with default configuration.
 	issConfig := iss.DefaultConfig(nodeIds)
 	issProtocol, err := iss.New(t.NodeID(mirID), issConfig, newMirLogger(managerLog))
 	if err != nil {
 		return nil, xerrors.Errorf("could not instantiate ISS protocol module: %w", err)
 	}
 
-	app := NewApplication(reqStore)
+	app := NewApplication()
 
 	node, err := mir.NewNode(
 		t.NodeID(mirID),
 		&mir.NodeConfig{
 			Logger: newMirLogger(managerLog),
 		},
-		&modules.Modules{
-			Net:          net,
-			WAL:          wal,
-			RequestStore: reqStore,
-			Protocol:     issProtocol,
-			App:          app,
-			Crypto:       &mirCrypto.DummyCrypto{DummySig: []byte{0}},
-		})
+		map[t.ModuleID]modules.Module{
+			"net":    net,
+			"iss":    issProtocol,
+			"app":    app,
+			"crypto": mirCrypto.New(&mirCrypto.DummyCrypto{DummySig: []byte{0}}),
```

Comment:
Can we already use the new crypto implementation here?

Reply:
Fixed